test/conftest.py

```diff
@@ -30,7 +30,7 @@
 
 try:
     import mxnet as mx
-except ImportError:
```

Comment:
The import above can raise `OSError` as well, so I'm catching that too to turn off `mx`